### DISCUSSION TITLE
合成化學家有福了!!

### DISCUSSION THREAD

**Discussion by stan890425** [1101-GS4719-C/Python-Community-GS4719-C#635] [General]
化學家一班的工作是將我們常見或容易獲取的材料和成為我們的目標產物，
傳統是要靠經驗將分子一步一步拆解，
但有時候相似的分子不一定能用相似的方法合成，所以常常會走許多冤枉路。
感謝科技的幫忙現在可以藉由美國化學協會的Scifinder n 的 AI幫助我們逆合成，
甚至可以先預估出你的預計花費及產率。
真的是電腦也會選花生ㄟ~~~

https://www.cas.org/solutions/cas-scifinder-discovery-platform/cas-scifinder/retrosynthesis-planning

**Comment by Jay-722** [comment]
反正我對化學是恨透了

**Comment by henrywen547** [comment]
我對有機無機的印象只停留在高中
現在只記得當時背了很多反應式
